apache/trafficcontrol
# TP: adds a better view for comparing/managing params of 2 profiles

## What does this PR (Pull Request) do?

Provides the ability to easily compare parameters across 2 profiles and easily add or remove parameters from each profile.

A blueprint is available for this new feature: https://github.com/apache/trafficcontrol/blob/master/blueprints/profile-param-compare-manage.md

- [x] This PR fixes #2391 

## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?
- Launch TP
- Navigate to https://tp.domain.com/#!/profiles
- Select "Compare Profiles" under the More button
- Select 2 profiles to compare
- Ensure each profile has the proper parameters "checked". (you can double check by checking the parameters for the profile at https://tp.domain.com/#!/profiles/{profile_id}/parameters)
- add or remove parameters from each profile; click "Update"
- ensure the proper parameters are assigned to each profile after confirmation.

## The following criteria are ALL met by this PR
